foia_hub/management/commands/load_agency_contacts.py

```diff
@@ -22,9 +15,33 @@ class Command(BaseCommand):
     """
 
     def handle(self, *args, **options):
-        try:
+        if len(args) > 0:
             yaml_folder = args[0]
-        except IndexError:
-            yaml_folder = _get_yaml_folder()
+        else:
+            yaml_folder = download_data()
 
+        print("Data directory: %s" % yaml_folder)
```

Comment:
Management commands should use self.stdout.write() and self.stderr.write() instead print(). See: https://docs.djangoproject.com/en/1.7/howto/custom-management-commands/#module-django.core.management